content/sensu-core/2.0/_index.md

```diff
@@ -17,25 +17,75 @@ layout: "single"
 Sensu is the open source monitoring event pipeline built to reduce operator burden and make developers and business owners happy.
 Started in 2011, Sensu’s flexible approach solves the challenges of monitoring hybrid-cloud and containerized infrastructures with scalable, automated workflows and integrations with tools you already use.
 
-Sensu 2.0 is a new framework, written in Go and designed from the ground up to be more portable, easier and faster to deploy, and (even more) friendly to containerized and ephemeral environments.
-But above all, it was designed to support all of the features and functionality you’ve come to know and love about Sensu.
+Sensu 2.0 is a new platform, written in Go and designed from the ground up to be more portable, easier and faster to deploy, and (even more) friendly to containerized and ephemeral environments.
```

Comment:
I think this should be containerized _with_ ephemeral environments. Not sure, but `and` doesn't read right.